Build/Build.cs

```diff
@@ -302,9 +302,48 @@ from framework in supportedFrameworks
             ReportTestOutcome(projects.Select(p => $"*{p.Name}*.trx").ToArray());
         });
 
+    Target TestingPlatformFrameworks => _ => _
+        .DependsOn(Compile)
+        .OnlyWhenDynamic(() => RunAllTargets || HasSourceChanges)
+        .Executes(() =>
+        {
+            Project[] projects =
+            [
+                Solution.TestFrameworks.TUnit_Specs
+            ];
+
+            var testCombinations =
+                from project in projects
+                let frameworks = project.GetTargetFrameworks()
+                from framework in frameworks
+                select new { project, framework };
+
+            DotNetTest(s => s
+                .SetConfiguration(Configuration.Debug)
+                .SetProcessEnvironmentVariable("DOTNET_CLI_UI_LANGUAGE", "en-US")
+                .EnableNoBuild()
+                .CombineWith(
+                    testCombinations,
+                    (settings, v) => settings
+                        .SetProjectFile(v.project)
+                        .SetFramework(v.framework)
+                        .SetProcessArgumentConfigurator(args => args
```

Comment:
TUnit is built on top of the newer Testing platform instead of VSTest. This means that VSTest CLI args aren't the same, so I needed to create a new target that uses different flags.

Reply:
You could also contribute this new testing platform's arguments to [Nuke](https://github.com/nuke-build/nuke) itself, if you want to ;)